src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -82,7 +83,7 @@ static bool processSubscriptions
                                          trigs->attrL,
                                          mapSubId,
                                          trigs->reference,
-                                         trigs->format,
+                                         JSON,
```

Comment:
However, I've just realized... is "JSON" correct? Or it should be "NGSI_V1_JSON"?

Actually, this lead to a new question... Is "JSON" stills working as "symbol" in the code? After the refactoring from `Format` enumeration to `NotificationFormat` I have said that "JSON" would no be longer valid...